src/dvb.c

```diff
@@ -188,15 +196,8 @@ int detect_dvb_parameters(char *s, transponder *tp)
 
 	tp->pids = tp->apids = tp->dpids = tp->x_pmt = NULL;
 
-	while (*s > 0 && *s != '?')
-		s++;
-
-	if (*s == 0)
-		LOG_AND_RETURN(0, "no ? found in URL");
-
-	s++;
-	if (strstr(s, "freq="))
-		init_dvb_parameters(tp);
+//	if (strstr(s, "freq="))
```

Comment:
why do we fail to initialize the dvb_parameters here ?

Reply:
https://github.com/catalinii/minisatip/blob/9af1d6560bc7e248860b13d980cc91dc3bdfccd2/src/dvb.c#L280
this will zero out the most of the tp params that set before to -1

Reply:
I did some tests again and it seem to make no difference if it's in or not.